owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -237,3 +244,44 @@ fun Activity.hideSoftKeyboard() {
         )
     }
 }
+
+fun Activity.checkPasscodeEnforced(securityEnforced: ISecurityEnforced) {
+    val sharedPreferencesProvider = SharedPreferencesProviderImpl(this)
+
+    val passcodeEnforced = this.resources.getBoolean(R.bool.passcode_enforced)
```

Comment:
If you are using this variable as a flag for the security lock enforcement, maybe you can call it `lockEnforced` instead of `passcodeEnforced`, it can lead to confusion (I thought there may be another variable for pattern enforcement)

Reply:
Good catch! 👍🏻